src/components/DotNavigation/DotNavigation.tsx

```diff
@@ -3,7 +3,10 @@ import styles from './DotNavigation.st.css';
 import { RadioButton, RadioButtonKeyDownEvent } from 'wix-ui-core/radio-button';
 import { DotNavigationDataKeys, DotNavigationDataHooks } from './dataHooks';
 import classNames from 'classnames';
-import * as _ from 'lodash';
+
+const isNan = require('lodash/isNan');
```

Comment:
Note that the build is breaking.
Need to change this:
```suggestion
const isNaN = require('lodash/isNaN');
```